src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java

```diff
@@ -294,6 +294,16 @@ public final String getLine(int index) {
         return context.get().fileContents.getLine(index);
     }
 
+    /**
+     * Returns the line associated with the tree.
```

Comment:
I see that this is consistent with the other `getLine...` methods, but maybe we can do better here. How about something like "Returns code point representation of file text from given line number."?

Reply:
done

Reply:
Resolved.